frontend/public/style/_overrides.scss

```diff
@@ -387,7 +387,7 @@ kbd {
 }
 
 // override .co-m-table-grid [class*='col-'], .co-m-table-grid .row
-.pf-c-table.pf-c-virtualized tr > td {
+.pf-c-table.pf-c-virtualized tr > td, .co-m-table-grid .pf-c-table [class*='col-'] {
```

Comment:
can you explain the need for this change? i thought `co-m-table-grid` was deprecated and we weren't using it for chargeback

Reply:
yes, removed old code.